app/Helpers/database/leaderboard.php

```diff
@@ -537,84 +539,28 @@ function UploadNewLeaderboard(
             return false;
         }
 
-        $query = "SELECT DisplayOrder FROM LeaderboardDef WHERE ID='$idInOut'";
-        $dbResult = s_mysql_query($query);
-        if ($dbResult !== false && mysqli_num_rows($dbResult) == 1) {
-            $data = mysqli_fetch_assoc($dbResult);
-            $displayOrder = $data['DisplayOrder'];
-            $displayOrder = (int) $displayOrder;
+        $foundLeaderboard = Leaderboard::find($idInOut);
+        if ($foundLeaderboard) {
+            $displayOrder = $foundLeaderboard->order_column;
+            $data['DisplayOrder'] = $displayOrder;
         }
     }
 
-    if (!submitLBData($author, $idInOut, $mem, $title, $desc, $format, $lowerIsBetter, $displayOrder)) {
+    if (!submitLBData($authorUsername, $idInOut, $mem, $title, $desc, $format, $lowerIsBetter, $displayOrder)) {
         $errorOut = "Internal error updating leaderboard.";
 
         return false;
     }
 
-    if ($originalAuthor != '') {
+    if (isset($originalAuthor)) {
         addArticleComment("Server", ArticleType::Leaderboard, $idInOut,
-            "$author edited this leaderboard.", $author
+            "{$authorModel->display_name} edited this leaderboard.", $authorModel->username
```

Comment:
Not sure if it matters, but edits from Connect use this message
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/1624e6eb-56ec-49b6-8f7a-f44217632231)

And edits from the webpage use this one
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/ca6d4d34-ee89-4aab-9601-504687b7d580)

Reply:
Probably not a huge deal if our long-term goal is to move over to the activitylog, but no harm in getting these consistent. Changed the webpage edits to match "edited this leaderboard".